guides/common/modules/proc_installing-puppet-agent-during-host-provisioning.adoc

```diff
@@ -49,11 +53,12 @@ ifndef::katello,orcharhino,satellite[]
 +
 Select the *boolean* type and set the value to `true`.
 endif::[]
-ifdef::katello,orcharhino,satellite[]
-. Enable the Puppet agent using a host parameter in global parameters, a host group, or for a single host.
-Add a host parameter named `enable-puppet7`, select the *boolean* type, and set the value to `true`.
-endif::[]
+. If you want to deploy the Puppet agent for a non-integrated Puppet setup, configure the following host parameters in global parameters or a host group:
+* Add a host parameter named `puppet_server`, select the *string* type, and set the value to the hostname of your Puppet server, such as `puppet.example.com`.
+* Optional: Add a host parameter named `puppet_ca_server`, select the *string* type, and set the value to the hostname of your Puppet CA server, such as `puppet-ca.example.com`.
+If `puppet_ca_server` is not set, the Puppet agent will use the same server as `puppet_server`.
+* Optional: Add a host parameter named `puppet_environment`, select the *string* type, and set the value to the Puppet environment you want the host to use.
 ifdef::katello,orcharhino,satellite[]
 . Ensure your host has access to the Puppet agent packages from {ProjectServer} by using an appropriate activation key.
 endif::[]
-. Ensure that you assign a Puppet environment, Puppet server and Puppet CA server to your host.
\ No newline at end of file
+. If you use an integrated setup, ensure that you assign a Puppet server, Puppet CA server, and Puppet environment to your host.
```

Comment:
This implies that `pupper_server` has to be created for both non-integrated and integrated setup, thus it should be extracted from the step for non-integrated setup where it is currently in. Additionally, I feel like this should maybe be a note under the non-integrated step rather than its own step.